### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231201

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1149] [Announcements]
本次更新只包含Rockchip机型

# 设备
* r4s：
  1. 跟 r4se 拆分 uboot，以修复 22.03 版本以来全新刷机无法启动的问题  #984 #1106
* rk35xx：
  1. NPU 驱动拆分成模块
  2. HDMI 驱动拆分成模块，但未经过足够设备测试，所以未集成。对于极客或开发者，如果自己编译固件，可以选中 kmod-drm-rockchip-rk35xx 以及 kmod-drm-rockchip-rk35xx-hdmitx 来启用 HDMI 输出（除R6S/R6C外可能还需要调整设备树文件）。

**Comment by mustssr** [comment]
rk35xx的NPU有什么用？NAT加速？

**Reply by jjm2473** [reply]
我们没用上这个功能，自己查查rknpu有什么使用场景

**Reply by jjm2473** [reply]
可能某些人会用来跑AI应用
